pkgs/development/tools/alire/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, fetchFromGitHub, gnat, gprbuild }:
+
+stdenv.mkDerivation {
+  pname = "alire";
+  version = "1.3-dev";
+  src = fetchFromGitHub {
+    owner = "alire-project";
+    repo = "alire";
+    rev = "8f44a810";
+    hash = "sha256-kYntEMeVPo1+gRI+Q/6qfSyICXNf8yLT+LcbuyeRu/E=";
+    fetchSubmodules = true;
+  };
+
+  OS = if stdenv.isDarwin then "macOS" else null;
+
+  nativeBuildInputs = [ gnat gprbuild ];
+
+  buildPhase = ''
+    runHook preBuild
+    gprbuild -j0 -P alr_env
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    gprinstall --mode=usage -P alr_env -p -r --prefix=$out
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/alire-project/alire";
+    description = "ALIRE: Ada LIbrary REpository.";
```

Comment:
Please remove `ALIRE:` and a period.

> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description